enet server test/enet server test.cpp

```diff
@@ -2147,6 +2148,9 @@ int _tmain(int argc, _TCHAR* argv[])
 			}
 
 			event.peer->data = new PlayerInfo;
+			/* Get the string ip from peer */
+			char clientConnection[32];
+			((PlayerInfo*)(peer->data))->charIP = enet_address_get_host_ip(&peer->address, clientConnection, 32);
```

Comment:
`enet_address_get_host_ip` doesn't return text (it returns if function failed or not)
```suggestion
			enet_address_get_host_ip(&peer->address, clientConnection, 16);
			((PlayerInfo*)(peer->data))->charIP = clientConnection;
```